CONTRIBUTING.rst

```diff
@@ -74,3 +74,24 @@ Before you submit a pull request, check that it meets these guidelines:
 * New classes, functions, etc have docstrings.
 * New code has comments.
 * Code style and file structure is similar to the rest of the project.
+* You have run the `black` code formatter.
+
+Releasing CumulusCI
+-------------------
+
+It's easy to release a version of CumulusCI to GitHub and PyPi! First, create a new branch for your version:
+
+    $ git checkout -b feature/newversion
+
+After commiting any updates to HISTORY.rst and the docs, bump the version:
+
+    $ bump2version patch
+    $ git push -u origin HEAD
+
+Open a Pull Request on GitHub and request approval from another commiter. Once your PR has been merged, you can create the release tag and push an .egg to PyPi via twine.
```

Comment:
committer, PyPI, "push the artifacts" instead of "push an .egg" (there's a wheel too)

Reply:
Can we link to instructions for setting up your ~/.pypirc?